src/lib/notificationWorker.js

```diff
@@ -117,25 +111,25 @@ class NotificationWorker {
           ]
         }
       } else if (caseInstance.state === 'rejected') {
-        transfer.cancellation_condition_fulfillment = stateAttestationSigned
+        action.cancellation_condition_fulfillment = stateAttestationSigned;
       } else {
         retry = false
         throw new Error('Tried to send notification for a case that is not yet finalized')
       }
 
-      const result = yield request(notification.transfer, {
+      const result = yield request(notification.action, {
         method: 'put',
         json: true,
-        body: transfer
-      })
+        body: action
+      });
```

Comment:
@naoitoi Thanks for merging #12 - can you rebase please and fix the linting issues? You can run lint locally via `npm run lint` and it'll tell you any remaining issues.